src/System.Device.Gpio.Tests/RaspberryPi4DriverTests.Linux.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio.Drivers;
+using System.Threading;
+using Xunit;
+
+namespace System.Device.Gpio.Tests
+{
+    public class RaspberryPi4DriverTests : GpioControllerTestBase
```

Comment:
This won't work until we actually have physical Raspberry Pi 4 devices on the lab in a queue. We will have to figure out how to test this before merging.

Reply:
I got one Pi4 (1GB) but I haven't had time to set it up yet